Packer/ArkCase-FOIA/startup.sh

```diff
@@ -114,3 +119,14 @@ systemctl start alfresco
 systemctl start config-server
 systemctl start arkcase
 systemctl start haproxy
+
+## Wait for ArkCase to become available on /arkcase
+timeout 600 bash -c 'while [[ "$(curl --insecure -s -o /dev/null -w ''%{http_code}'' https://localhost/arkcase/login)" != "200" ]]; do sleep 5; done'
```

Comment:
Is 600 seconds enough on first start? Or this is not the first start?

Reply:
It is not a first start. The first is in the packer process where AMI is built